lib/pushmi_pullyu/cli.rb

```diff
@@ -59,13 +59,13 @@ def configure_rollbar
     Rollbar.configure do |config|
       config.enabled = false unless options[:rollbar][:token].present?
       config.access_token = options[:rollbar][:token]
-
+      config.exception_level_filters['IOError'] = 'ignore'
```

Comment:
Maybe use their provided syntax? If we decide to filter another exception in the future, someone doesn't just come in and copy and paste this line, they can add it to the merge call:

```
config.exception_level_filters.merge!({
  'IOError' => 'ignore'
})
```